recipes/json-e/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "json-e" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: d2914f785d93ecc4f0b2ad6e3f2791f33327eaa740a3c4917d68a9a485dd282d
+    folder: dist
+  - url: https://github.com/taskcluster/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
I think you just need the ``github`` tarballs right?
You can delete the package being download from pypi and install everything from the tarballs

Reply:
Yep, building/installing the sdist works fine.